login.py

```diff
@@ -0,0 +1,104 @@
+import requests
+import json
+import re
+from urllib import parse
+
+requests.packages.urllib3.disable_warnings(requests.packages.urllib3.exceptions.InsecureRequestWarning)
+
+OIDC_LOGIN_URL="http://localhost:5000/dp/v1/login"
+USERS_CONFIGURATION_PATH="oauth/users.json"
+CLIENTS_CONFIGURATION_PATH="oauth/clients-config.json"
+
+f = open(USERS_CONFIGURATION_PATH)
+data = json.load(f) 
```

Comment:
This script can't be imported by another module. Can we convert it to an executable module? (https://docs.python.org/3/tutorial/modules.html#executing-modules-as-scripts)